src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+const VeteranInformationComponent = props => {
+  let dateOfBirthFormatted = '';
+  let genderFull = '';
+  if (props.user && props.user.dob) {
+    dateOfBirthFormatted = moment(props.user.dob).format('MMMM Do YYYY');
+  }
+  if (props.user && props.user.gender === 'M') {
+    genderFull = 'Male';
+  } else if (props.user && props.user.gender === 'F') {
+    genderFull = 'Female';
+  }
+  const alertContent = (
+    <>
+      <p>
```

Comment:
I wonder if this would be a good candidate for use of a definition list. 🤔

Reply:
Updated this